composer.json

```diff
@@ -12,7 +12,8 @@
         }
     ],
     "require": {
-        "yiisoft/yii2": "*"
+        "yiisoft/yii2": "*",
+        "ext-bcmath": "*"
```

Comment:
hello @uldisn , do we need to add this to composer?

Reply:
I don't see any of these functions used anywhere.  Is it needed?

Reply:
https://github.com/cornernote/yii2-returnurl/blob/master/src/ReturnUrl.php#L147
```php
$hash = bcadd(sprintf('%u', crc32($data)), 0x100000000);
```